common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -536,10 +529,7 @@
                         } else {
                             if (state.isTouch) {
                                 self.subtitlesEl.find('.subtitles-menu').html(
-                                    gettext(
-                                        '<li>Transcript will be displayed when ' +
-                                        'you start playing the video.</li>'
-                                    )
+                                    '<li>' + gettext('Transcript will be displayed when you start playing the video.') + '</li>' // jshint ignore:line
```

Comment:
Yeah, the concatenation of HTML and translated text needs to be done with HTMLUtils per the safe template best practices. @clrux this is being done in a different PR?